### DISCUSSION TITLE
Suggestion for .bootenv

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#290] [General]
This is a small change I made to my copy of .bootenv just to give me an idea of what it is doing:

![image](https://github.com/ZOSOpenTools/meta/assets/42328411/725123b4-336b-4804-a25e-65473ce08f96)

Another suggestion is to replace the list of subdirectories (ported tools) with a dynamically generated list.

**Comment by v1gnesh** [comment]
Hey Lionel, `zopen-importenvs` picks stuff up dynamically.
https://github.com/ZOSOpenTools/meta/blob/a4ae250f55ef539e368186f8c778f565ae779105/bin/zopen-importenvs#L11

**Reply by lbdyck** [reply]
it does but it also appears to be for bash - what about those just using /bin/sh ?

**Reply by lbdyck** [reply]
Plus looking for something for the user rather than the developer.

**Reply by v1gnesh** [reply]
Ahhh, right ok :(

**Comment by IgorTodorovskiIBM** [comment]
Thanks @lbdyck, I believe this is the code in question: https://github.com/ZOSOpenTools/meta/blob/main/tools/src/createbootenv.c#L40 . 

@sabitor, since you're looking at zopen-setup, this might be an easy one for you once you get the compiler issues sorted.

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM and @lbdyck is this solved?
